hotline/species/templates/species/category_create_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h2>Create category</h2>
+    
+    <form action="" method="post">
+        {% csrf_token %}
+        {{ form.name|bootstrap }}
+    <input type="submit" class="btn btn-primary" value="Create" />

```

Comment:
It looks like `hotline/species/templates/species/category_create_form.html` and `hotline/species/templates/species/category_detail_form.html` are exactly the same, except for one line of text. Can they be refactored?

Also, for non-reusable apps, we like to put templates in the templates directory for the `project`, not the app.